extensions/toggl-track/README.md

```diff
@@ -0,0 +1,41 @@
+# Raycast Toggl Track Extension
+
+## Getting started
+
+To use the extension you need to submit your API Token. This can be found on the bottom of your profile in toggle track: https://track.toggl.com/profile
```

Comment:
```suggestion
To use this extension, you need your Toggl Track API Token. This can be found on the bottom of your profile in toggle Track: https://track.toggl.com/profile
```